x/projects/client/cli/tx_add_keys.go

```diff
@@ -15,18 +15,18 @@ import (
 
 var _ = strconv.Itoa(0)
 
-func CmdAddProjectKeys() *cobra.Command {
+func CmdAddKeys() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "add-project-keys [project-id] [optional: project-keys-file-path]",
+		Use:   "add-keys [project-id] [optional: project-keys-file-path]",
 		Short: "Add developer/admin keys to an existing project",
-		Long: `The add-project-keys command allows the project admin to add new project keys (admin/developer) to the project.
+		Long: `The add-keys command allows the project admin to add new project keys (admin/developer) to the project.
 		To add the keys you can optionally provide a YAML file of the new project keys (see example in cookbook/project/example_project_keys.yml).
```

Comment:
The path to the example_project_keys.yml file is old. The right path is cookbook/projects/example_project_keys.yml (note the s in projects)